Source/DafnyDriver/InputOptions.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Boogie;
+
+namespace Microsoft.Dafny; 
+
+public class InputsOption : CommandLineOption<IEnumerable<string>> {
+  public static readonly InputsOption Instance = new();
+  public override object DefaultValue => new List<string>();
+  public override string LongName => "input";
+  public override string ArgumentName => "file";
+  public override string Description => "Specify additional input files.";
```

Comment:
"(use this repeatedly if you want to include multiple files)" the help already indicates this by placing a `[]` at the end of the description, although that isn't very clear. We could also customise the help to make the `[]` message more explicit.

Reply:
I'm not familiar with the [] syntax. Where is it described? In any case, if you use "files" instead of "file", I as a user would be confused because that would indicate I could do something like `--input file1.dfy,file2.dfy`, whereas if I understand `--input`, it should be one file at a time, correct?

Reply:
> I'm not familiar with the [] syntax. Where is it described?

I don't know, but you can see it if you do `dafny build --help`